tools/ledger/src/main/java/org/apache/bookkeeper/tools/cli/commands/BookieCommandGroup.java

```diff
@@ -40,6 +41,7 @@
         .withParent(BKCtl.NAME)
         .withCategory(CATEGORY_INFRA_SERVICE)
         .addCommand(new LastMarkCommand())
+        .addCommand(new InfoCommand())
```

Comment:
Actually I just realized that `InfoCommand` is displaying all the informations for the whole cluster, not a single bookie. so I would suggest adding this command as part of `bookies` group. (https://github.com/apache/bookkeeper/blob/master/tools/ledger/src/main/java/org/apache/bookkeeper/tools/cli/commands/BookiesCommandGroup.java)

So you might consider adding the command to this location: https://github.com/apache/bookkeeper/tree/master/bookkeeper-server/src/main/java/org/apache/bookkeeper/tools/cli/commands/bookies